portfolio.html

```diff
@@ -14,19 +15,20 @@ <h2 class="contentHeadings"><i class="fas fa-rocket"></i> Career Profile</h2>
 					<div class="profileBody"> I decided tyo learn to code after years of frustrating experience with charity funding, instead of using all my energy to secure funding for projects that do good I decided to learn to code so that I could actually design and deliver digital projects that do good and that can be self sufficient. Designing solutions that don't require dependence on increasingly scarce resources.</div></section>
 		
 				<section class="experience"><h2 class="contentHeadings"><i class="fas fa-rocket"></i> Experience</h2>
-				<div class="experienceItemHeading"> The Bristol Bike Project
-					<div class="experienceItemBody"> am currently leading on a research project at Bristol Bike Workshop, examining the barriers to stopping young women accessing cycling in general and the community workshop offered at the project. I have secured funding from the School for Social Entrepreneurs, designed the research brief and am currently in progress of delivering a pilot programme and research programme.</div>
+						
+							<div class="experienceItemHeading"> The Bristol Bike Project
+									<div class="experienceItemBody"> am currently leading on a research project at Bristol Bike Workshop, examining the barriers to stopping young women accessing cycling in general and the community workshop offered at the project. I have secured funding from the School for Social Entrepreneurs, designed the research brief and am currently in progress of delivering a pilot programme and research programme.</div> </div>
 				
-						<div class="experienceItemHeading"> Outdoor People
-							<div class="experienceItemBody"> I  acted as a research consultant for Outdoor People, completing major pieces of research on behalf of Unilever's Outdoor Classroom Day campaign and Iceland. The research work we conducted went on to be debated in the Scottish parliament and I have given talks on the research at various conferences.</div>
-				</div>
+							<div class="experienceItemHeading"> Outdoor People
+								<div class="experienceItemBody"> I  acted as a research consultant for Outdoor People, completing major pieces of research on behalf of Unilever's Outdoor Classroom Day campaign and Iceland. The research work we conducted went on to be debated in the Scottish parliament and I have given talks on the research at various conferences.</div> </div>
+							
 		
 		
 				</section>
 		
 				<section class="projects"><h2 class="contentHeadings"><i class="fas fa-rocket"></i> Projects</h2>
```

Comment:
Few cases where tags have been put inline. Are these to avoid new lines being created? If so, could possibly wrap it in a span tag below?